### PR TITLE
mixpanel: record workspace for breakpoint

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: src/ui/components/DevTools.tsx [hunk 1]
```diff
@@ -71,12 +72,17 @@ function DevTools({
...
     }
+    // TODO(dmiller): maybe move the view recording event here
   }, [loading, userId, sessionId]);
 
   useEffect(() => {
     if (title) {
       document.title = `${title} - Replay`;
     }
+    if (recording?.workspace) {
+      debugger;
```
Comment (loganfsmyth) [inline src/ui/components/DevTools.tsx:83]: :shipit:

**Review by loganfsmyth — APPROVED** [review]
Seems like a reasonable approach to me pending TODO cleanup and the `debugger`.

File: src/devtools/client/debugger/src/actions/breakpoints/modify.js [hunk 2]
```diff
@@ -78,15 +82,28 @@ export function enableBreakpoint(cx, initialBreakpoint) {
...
+  }
+  mixpanel.track("breakpoint", context);
+}
+
 export function addBreakpoint(
   cx,
   initialLocation,
   options = {},
   disabled = false,
+  shouldTrack = false,
```
Comment (jasonLaster) [inline src/devtools/client/debugger/src/actions/breakpoints/modify.js:98]: this makes me nervous that somewhere there is a callsite that expects `shouldCancel`  to be the 5th param...

File: src/ui/utils/apolloClient.ts [hunk 3]
```diff
@@ -27,9 +27,9 @@ export function createApolloClient(token: string | undefined, recordingId: strin
   const httpLink = createHttpLink(token, recordingId);
   const errorLink = onError(({ graphQLErrors, networkError }) => {
     if (graphQLErrors) {
-      console.log(`[Apollo GraphQL error]: ${graphQLErrors}`);
+      console.log("[Apollo GraphQL error]:", graphQLErrors);
     } else if (networkError) {
-      console.log(`[Apollo Network error]: ${networkError}`);
+      console.log("[Apollo Network error]:", networkError);
```
Comment (jasonLaster) [inline src/ui/utils/apolloClient.ts:32]: not a big deal, but i'm curious why you made this change.

Comment (jazzdan) [reply src/ui/utils/apolloClient.ts:32]: This was a drive by fix. Without this, the log shows up as `[Apollo Network error]: [object Object]` which isn't super helpful

File: src/devtools/client/debugger/src/actions/breakpoints/modify.js (hunk 2 above)
Comment (jazzdan) [reply src/devtools/client/debugger/src/actions/breakpoints/modify.js:98]: Good to be nervous, I just checked again and all the callsites seem good.

**Comment by bhackett1024** [comment]
I backed this out, see #2428

**Pull request merged**